src/tests/banners/DefaultContributionsBannerTest.ts

```diff
@@ -1,55 +1,24 @@
-import fetch from 'node-fetch';
-import {
-    BannerTestGenerator,
-    BannerContent,
-    BannerTest,
-    BannerTargeting,
-    BannerPageTracking,
-} from '../../types/BannerTypes';
+import { BannerPageTracking, BannerTargeting, BannerTest } from '../../types/BannerTypes';
+import { DefaultBannerContent } from './DefaultContributionsBannerContent';
 
 export const DefaultContributionsBannerPath = 'contributions-banner.js';
 
-const defaultBannerContentUrl =
-    'https://interactive.guim.co.uk/docsdata/1CIHCoe87hyPHosXx1pYeVUoohvmIqh9cC_kNlV-CMHQ.json';
-
-const DefaultContributionsBanner = (bannerContent: BannerContent): BannerTest => {
-    return {
-        name: 'DefaultContributionsBanner',
-        bannerChannel: 'contributions',
-        testAudience: 'AllNonSupporters',
-        canRun: (targeting: BannerTargeting, pageTracking: BannerPageTracking): boolean =>
-            // Do not serve to frontend for now
-            pageTracking.clientName === 'dcr',
-        minPageViews: 2,
-        variants: [
-            {
-                name: 'control',
-                modulePath: DefaultContributionsBannerPath,
-                moduleName: 'ContributionsBanner',
-                bannerContent: bannerContent,
-                componentType: 'ACQUISITIONS_ENGAGEMENT_BANNER',
-            },
-        ],
-    };
-};
-
-const defaultBannerTest = (bannerContent: BannerContent): BannerTest => {
-    return DefaultContributionsBanner(bannerContent);
+export const DefaultContributionsBanner: BannerTest = {
+    name: 'DefaultContributionsBanner',
+    bannerChannel: 'contributions',
+    testAudience: 'AllNonSupporters',
+    canRun: (targeting: BannerTargeting, pageTracking: BannerPageTracking) => {
+        // Do not serve to frontend for now
+        return pageTracking.clientName === 'dcr';
```

Comment:
this condition can be removed now, frontend fetches all banners from here

Reply:
Good point!